packages/amplify-cli/src/__tests__/init-steps/s0-analyzeProject.test.ts

```diff
@@ -75,4 +76,35 @@ describe('analyzeProject', () => {
     await analyzeProject(mockContext);
     expect(mockContext.exeInfo.isNewEnv).toBe(true);
   });
+
+  it('throws helpful error message when running subsequent init -y commands', async () => {
+    const emitErrorMock = jest.fn();
+
+    mockContext.amplify = {
+      confirmPrompt: jest.fn(),
+      getAllEnvs: jest.fn().mockReturnValue(['dev']),
+      pathManager: {
+        getProjectConfigFilePath: jest.fn().mockReturnValue('mockDirPath'),
+      },
+    };
+    mockContext.exeInfo = {
+      isNewProject: false,
+    };
+    mockContext.exeInfo.inputParams = {
+      amplify: {},
+      yes: true,
+    };
+    mockContext.usageData = {
+      emitError: emitErrorMock,
+    };
+
+    jest.spyOn(fs, 'existsSync').mockReturnValue(true);
```

Comment:
Mocked the bare minimum to replicate the situation where `amplify-dev init -y` was run the first time.